blaze/compute/numpy.py

```diff
@@ -122,12 +150,12 @@ def compute_up(t, x, **kwargs):
 
 @dispatch(Not, np.ndarray)
 def compute_up(t, x, **kwargs):
-    return ~x
+    return np.logical_not(x)
```

Comment:
Right, it was technically incorrect then. Blaze's `Not` operation is only defined for boolean datashaped things